frontend/src/components/BackLink.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faArrowLeft } from '@fortawesome/free-solid-svg-icons';
+import { Link } from 'react-router-dom';
+import colors from '../colors';
+import './BackLink.scss';
+
+export default function BackLink({
+  to, children, iconClasses, linkClasses,
+}) {
+  return (
+    <>
+      <FontAwesomeIcon className={`margin-right-1 ${iconClasses}`} data-testid="back-link-icon" color={colors.ttahubMediumBlue} icon={faArrowLeft} />
```

Comment:
I didn't go about replacing this everywhere that it lives right now, but it is something we can work on as we go

Reply:
Probably want to create a ticket for this so we don't forget.